tensorflow_datasets/scripts/download_and_prepare.py

```diff
@@ -156,6 +156,13 @@ def import_modules(modules):
   for module in modules:
     importlib.import_module(module)
 
+def _get_version(builder_name, version = None):
```

Comment:
We should not duplicate logic. This code is already present in `tfds.builder` as `tfds.builder('ds:version')` works.
Could you find another way ?

Reply:
For instance, you could check that passing `tfds.builder(name, data_dir=FLAGS.data_dir, **version_kwargs)` works.
when `version_kwargs = {}` when `FLAGS.version is None`.